www/js/services.js

```diff
@@ -0,0 +1,38 @@
+angular.module('open311.services', [])
+.factory('API', function($http, $httpParamSerializer, $q) {
+
+    var recentCasesData = [];
+
+    var getRequests = function(lat, long) {
+      var deferred = $q.defer();
+
+      var params = $httpParamSerializer({
+          lat: lat,
+          long: long 
+      });
+
+      $http.get('https://seeclickfix.com/open311/v2/requests.json?' + params)
+        .then(function(response) {
+          recentCasesData = response.data;
+          deferred.resolve(response);
+        }, function(response) {
+          deferred.reject(response);
+        });
```

Comment:
On second thought, you don't actually need the error handler for now, since you're not really doing anything with the error.  I'd just remove it from the service, and let the UI code handle the error.